products/sle/main.pm

```diff
@@ -860,6 +860,10 @@ elsif (get_var('SYSTEMD_TESTSUITE')) {
 elsif (get_var('VALIDATE_PCM_PATTERN')) {
     load_public_cloud_patterns_validation_tests;
 }
+elsif (get_var('TERRAFORM')) {
+    boot_hdd_image;
+    loadtest "terraform/create_image";
+}
```

Comment:
Does this have to be SLE specific? You could move it to lib/main_common when you tested it also works for e.g. openSUSE Tumbleweed

Reply:
yes, will do, thanks for the suggestion

Reply:
done, now it's in main_common